linux/ubuntu/20.04/foxpass_setup.py

```diff
@@ -239,6 +291,12 @@ def fix_eic():
         os.system('systemctl daemon-reload')
 
 
+def check_sudo_passwd():
+    result = os.popen('passwd --status root').read().split(' ')
+    if result and result[1] != 'P':
+        sys.exit('Please set the password of the `root` user before enabling ldap sudoers. E.g sudo passwd root')
```

Comment:
why is this needed?

Reply:
Hi @aren , we cannot install `sudo-ldap` package if the root user's password is not set. 

This only happens in Ubuntu.